docs/pages/visual-test/Switch-dark.js

```diff
@@ -0,0 +1,18 @@
+// @flow strict
+import { type Node } from 'react';
+import { Flex, Switch, Box, ColorSchemeProvider } from 'gestalt';
+
+export default function Screenshot(): Node {
+  return (
+    <ColorSchemeProvider colorScheme="dark">
+      <Box color="white" display="inlineBlock" padding={1}>
+        <Flex gap={4}>
+          <Switch onChange={() => {}} id="1" switched />
```

Comment:
Can you add labels for these so we can tell what each represents? This will be very helpful when viewing these in VS Code

Reply:
I though about it... I doubted about Labels because Switch doesn't have built-in labels... So I decided to show the raw component. But I agree it's confusing.

Reply:
DONE